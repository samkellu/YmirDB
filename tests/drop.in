set a 123
snapshot
set b 22 343
snapshot
drop 5
drop 0
set c 22 a b 44
snapshot
list snapshots
drop 3
list snapshots
list entries
checkout 1
list entries
bye
