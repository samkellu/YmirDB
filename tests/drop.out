> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> no such snapshot

> no such snapshot

> ok

> saved as snapshot 3

> 3
2
1

> ok

> 2
1

> c [22 a b 44]
b [22 343]
a [123]

> ok

> a [123]

> bye
